tests/unit/capi/GEOSDistanceTest.cpp

```diff
@@ -9,6 +9,9 @@
 #include <cstdio>
 #include <cstdlib>
 #include <memory>
+#if defined(_MSC_VER)
+#  define _USE_MATH_DEFINES
+#endif
 #include <math.h>
```

Comment:
In other files we have this block:
```
#ifndef M_PI
#define M_PI        3.14159265358979323846
#endif
```

Please use the same (BTW, one such block is in platform.h.vc)